src/platform/Darwin/BleConnectionDelegateImpl.mm

```diff
@@ -103,6 +104,7 @@ - (id)initWithDiscriminator:(uint16_t)deviceDiscriminator
         _chipWorkQueue = chip::DeviceLayer::PlatformMgrImpl().GetWorkQueue();
         _timer = dispatch_source_create(DISPATCH_SOURCE_TYPE_TIMER, 0, 0, _workQueue);
         _centralManager = [CBCentralManager alloc];
+        _found = false;
```

Comment:
this variable never seem to be used.

Reply:
`_found` is another name for `self.found`, mostly.  So it's used.